plugins/scaffolder/src/components/ActionsPage/ActionsPage.tsx

```diff
@@ -68,6 +75,34 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+const ExamplesTable = (props: { examples: ActionExample[] }) => {
+  return (
+    <Grid container>
+      {props.examples.map(example => {
+        return (
+          <>
+            <Grid lg={3}>
```

Comment:
Now you don't have the `item` prop on the items instead.

Reply:
added the prop now.